charts/nextcloud/values.yaml

```
@@ -128,6 +128,7 @@ nextcloud:
   # Extra config files created in /var/www/html/config/
   # ref: https://docs.nextcloud.com/server/15/admin_manual/configuration_server/config_sample_php_parameters.html#multiple-config-php-file
   configs: {}
+    # defaultMode: 420
```

Please put a line without outcommanting (`if` and `with` would handle it as an empty/false)
```suggestion
    # set defaultMode for mounted configMaps (e.g. defaultMode: 420)
    defaultMode:
```